src/main/java/io/stargate/sgv2/jsonapi/service/resolver/update/TableUpdateResolver.java

```diff
@@ -97,6 +103,41 @@ public WithWarnings<UpdateValuesCQLClause> resolve(
               }));
     }
 
+    // Assignments list is empty meaning user does not specify any non-empty assignment operator
+    // Data API does not have task to do, error out.
+    if (assignments.isEmpty()) {
+      throw UpdateException.Code.MISSING_UPDATE_OPERATIONS.get(
+          errVars(
+              commandContext.schemaObject(),
+              map -> {
+                map.put("supportedUpdateOperations", errFmtJoin(supportedOperatorsStringList));
+              }));
+    }
+
+    // Duplicate column assignments is invalid
+    // e.g. {"update": {"$set": {"description": "123"},"$unset": {"description": "456"}}}
```

Comment:
Should we also handle subpaths? For example, `$set: { 'nested.path': 'test' }, $unset: { 'nested': 1 }`?

Another potential concern I had here was `$setOnInsert` and `$set` on the same field potentially being ok, but MongoDB also disallows that, so I think disallowing $setOnInsert and $set on the same field is reasonable.

Reply:
Thanks Val.
Currently only $set and $unset are supported in tables feature.

Reply:
and we do not have support for nested in tables, that may be how we handle maps and/or  UDT's